src/image/image_raw.rs

```diff
@@ -185,6 +185,7 @@ impl<'a> ImageRaw<'a, BinaryColor> {
     /// Returns an image with 0 x 0 pixel size if `width` is 0.
     //
     // MSRV: remove this function when const functions with trait bounds are supported
+    // TODO: What replaces this when removed?
```

Comment:
The regular `new` constructor. Depending on the context a type annotation might be required. `ImageRaw::<BinaryColor>::new(...)` should be a 1:1 replacement for `new_binary`.

Reply:
I should've read the doc comment... thanks for the explanation here.